File: Java/loops2.java
// Print first N even numbers
import java.util.Scanner;

public class loops2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter N: ");
        int n = sc.nextInt();

        for (int i = 1; i <= n; i++) {
            System.out.print(2 * i + " ");
        }
        System.out.println();
    }
}

File: Java_Practice/CPP/loops2.cpp
//first N even numbers
#include<iostream>
using namespace std;
int main()
{
    int n;
    cout<<"ENTER N : "<<endl;
    cin>>n;
    for(int i = 1 ; i<=n ; i++){
        cout<<"first N even numbers : "<<2*i<<endl;
    }
    return 0;
}